src/components/axes/timeAxis.ts

```diff
@@ -420,11 +450,11 @@ export module Axis {
       );
 
       var baselineOffset = 0;
-      for (i = 0; i < Math.max(tierConfigs.length, 1); ++i) {
+      for (var i = 0; i < Math.max(tierConfigs.length, 1); ++i) {
         var attr = this._generateBaselineAttrHash();
         attr["y1"] += (this.orient() === "bottom") ? baselineOffset : -baselineOffset;
         attr["y2"] = attr["y1"];
-        this._tierBaselines[i].attr(attr).style("visibility", "visible");
+        this._tierBaselines[i].attr(attr).style("visibility", "inherit");
```

Comment:
Suggestion from @jtlan . It makes more sense to enforce the visibility to inherit as that is the default behavior. It is also better practice because in the future the parent element might be set to `hidden` and this will break that change by enforcing visibility